<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiGDF</class>
 <widget class="QMainWindow" name="GuiGDF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GuiGDF</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ping">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>511</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="cmd">
     <item>
      <widget class="QPushButton" name="write">
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="read">
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="grep">
       <property name="text">
        <string>Grep</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ls">
       <property name="text">
        <string>List</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>341</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="status">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>341</width>
       <height>351</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>260</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="services">
     <item>
      <widget class="QPushButton" name="startAll">
       <property name="text">
        <string>StartAll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopAll">
       <property name="text">
        <string>StopAll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="statusAll">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="contentText">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>161</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>96</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="path"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fileName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Content</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="type">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>FS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hadoop</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hbase</string>
     </property>
    </item>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>ping</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>frame</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>contentText</zorder>
   <zorder>type</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="action_Close"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
